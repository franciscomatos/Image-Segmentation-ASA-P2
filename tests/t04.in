2 3

9 0 0
0 0 0

0 0 0
0 0 9

5 1
1 5

5 1 5
