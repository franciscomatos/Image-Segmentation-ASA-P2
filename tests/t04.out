3

C C P 
C P P 
